nomad-docker-wrapper

```diff
@@ -65,6 +65,11 @@ CONTAINER_ID=$(docker run -d \
   --name $NOMAD_DOCKER_CONTAINER_NAME \
   $NOMAD_DOCKER_COMMAND)
 
-echo "Waiting for container $NOMAD_DOCKER_CONTAINER_NAME to finish"
+if [ $? -ne 0 ]; then
+  echo "Failed to start $NOMAD_DOCKER_CONTAINER_NAME"
```

Comment:
There is no `$CONTAINER_ID` because in this branch `docker run` failed and therefore the container was not created.